frontend/src/features/idir/search/pages/IDIRSearchResultsDashboard.tsx

```diff
@@ -52,7 +53,8 @@ export const IDIRSearchResultsDashboard = memo(() => {
         id={`layout-tabpanel-search-results`}
         aria-labelledby={`layout-tab-search-results`}
       >
-        <IDIRSearchResults searchParams={searchFields} />
+        {searchFields?.searchEntity === 'companies' && <IDIRCompanySearchResults searchParams={searchFields} />}
```

Comment:
I personally prefer
`searchFields?.searchEntity === "companies" ? (<IDIRCompanySearchResults searchParams={searchFields} />) : null`
but I guess having this is fine as well

Reply:
I think you can reuse the SEARCH_ENTITIES enum inside idir/search/types/types.ts instead of the string "companies" and "permits"